fix(logs): (NR-371436) Clarify array support for logs (#20149)
newrelic/docs-website: src/content/docs/logs/log-api/introduction-log-api.mdx

      </td>

      <td>

>         Arrays will be _stringified_ and stored as strings. In the case of OpenTelemetry logs,

Reviewer: I don't think we should be using the verb _stringify_ in our pub docs, I'd rephrase it as: "Arrays will be **converted to text** ..."

>         Arrays will be _stringified_ and stored as strings. In the case of OpenTelemetry logs,

Reviewer 2: Suggestion:

```suggestion
        Arrays are converted to strings and stored as such. OpenTelemetry logs support
```

>         homogeneous arrays are supported. For more information see [arrays in OpenTelemetry](/docs/nrql/using-nrql/arrays-in-nrql/).

Reviewer 2: ```suggestion
        homogeneous arrays. For more information see [arrays in OpenTelemetry](/docs/nrql/using-nrql/arrays-in-nrql/).
```
